src/metadata.rs

```diff
@@ -120,3 +120,45 @@ impl<'a> SstFileMetaData<'a> {
         }
     }
 }
+
+pub struct LiveFiles {
+    inner: *const DBLivefiles,
```

Comment:
It this a memory leak? Might need to call xx_destroy in dtor.

Reply:
good point